mutator/statement/remove.go

```diff
@@ -100,6 +98,48 @@ func (m *MutatorRemoveStatement) Mutate(node ast.Node, changed chan bool) {
 	}
 }
 
+func createNoop(old ast.Stmt) ast.Stmt {
+	v := &idCollector{}
+	ast.Walk(v, old)
+	return v.generateStatement(old)
+}
+
+type idCollector struct {
```

Comment:
This issue should concern only "variables" so I think "idCollector" and the "Ids" field should be renamed.